### PR TITLE
ex5 starts

### PR DESCRIPTION
**Pull request by HassanHassouna**

### CODE REVIEW

**Comment by HassanHassouna** [comment]
Couldn't finish it all because of exams this week..

File: src/ex5/server/db/models/items.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    }
+  }
+  Items.init(
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+      },
+      ItemName: DataTypes.STRING,
+    },
```
Comment (amirbardugo) [inline src/ex5/server/db/models/items.js:21]: you didn't add status column

File: src/ex5/dist/main.js [hunk 2]
```diff
@@ -1,57 +1,56 @@
 class Main {
```
Comment (amirbardugo) [inline src/ex5/dist/main.js:1]: you have some bugs, i noticed that some time I add new task and I won't see it until I'll refresh

File: src/ex5/server/routes/api.js [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-    res.send(itemManager.getItems())
+router.get("/items", async (_, res) => {
+  res.send(await itemManager.getItems())
 })
 
-router.post('/item', async (req, res) => {
-    await itemManager.handleItem(req.body.item)
-    res.end()
+router.post("/item", async (req, res) => {
+  await itemManager.handleItem(req.body.item)
```
Comment (amirbardugo) [inline src/ex5/server/routes/api.js:10]: the router should just mention the controller action, the controller should take care of body and params seperation

File: src/ex5/server/services/item_manager.js [hunk 4]
```diff
@@ -1,53 +1,70 @@
...
-            pokemons.forEach(this.addPokemonItem);
-        } catch (error) {
-            console.error(error)
-            this.addItem(`Failed to fetch pokemon with this input: ${inputValue}`)
-        }
+  addPokemonItem = (pokemon) => {
+    Items.create({
+      ItemName: `Catch ${pokemon.name}`,
+      id: pokemon.id,
+    })
```
Comment (amirbardugo) [inline src/ex5/server/services/item_manager.js:34]: do you really want the item id to be the pokemon id? 
what would happen if I have regular item with id 1 and then I'll add pokemon id 1?

File: src/ex5/dist/main.js (hunk 2 above)
Comment (amirbardugo) [reply src/ex5/dist/main.js:1]: another issue is with the id's, they are not really uniq right now which cause problems in deletions etc..

**Pull request merged**